libr/bin/format/mach0/mach0.c

```diff
@@ -1199,23 +1199,22 @@ static int parse_function_starts(struct MACH0_(obj_t) *bin, ut64 off) {
 
 static int parse_dylib(struct MACH0_(obj_t) *bin, ut64 off) {
 	struct dylib_command dl;
-	int lib, len;
+	int len;
 	ut8 sdl[sizeof (struct dylib_command)] = {0};
 
 	if (off > bin->size || off + sizeof (struct dylib_command) > bin->size) {
 		return false;
 	}
-	lib = bin->nlibs - 1;
 
-	void *relibs = realloc (bin->libs, bin->nlibs * R_BIN_MACH0_STRING_LENGTH);
-	if (!relibs) {
-		r_sys_perror ("realloc (libs)");
+	char *lib = calloc (1, R_BIN_MACH0_STRING_LENGTH);
```

Comment:
better to hold this thing in the stack. its just 256 bytes. `char lib[256]` and then use strdup() when appending it to the list/vector. otherwise we are wasting memory and doing unnecessary allocations